### ISSUE TITLE
Neovim: Error when moved selected lines (Visual mode)

### ISSUE THREAD

**Issue by AlexMasterov** [axelf4/vim-strip-trailing-whitespace#4]
Hello.
Using this keybindings (`<<`, `>>`) on _any_ text:

```vim
" Alt-[hl]: move selected lines
xnoremap <A-h> <'[V']
xnoremap <A-l> >'[V']
```
I get the error:
```sh
1 lie >ed 1 time
Error executing lua callback: [string ":lua"]:1: Vim(call):E16: Invalid range: call s:RemoveRange(a:start, a:end - 1)
```
> NVIM v0.5.0-330-g71ee46acc

> VIM 8.2 (p150) — works fine

**Comment by axelf4** [comment]
Thanks for reporting, am able to reproduce with just `>>` (on the *last* line). I'm fairly certain this is a Neovim bug and have submitted a fix in neovim/neovim#11782. If that's the case I'll see about adding a workaround.

**Comment by AlexMasterov** [comment]
> NVIM v0.5.0-349-g486fa2189

Thanks for the fix 👍
